sections/sprint-3/goal-position-generation.tex

```diff
@@ -20,7 +20,8 @@ \subsection{Server-side}
 \subsection{Client-side}
 On the client-side, the new center points are received along with the goal size.
 Based on this data, the corners of the goal zones can be calculated.
-The corners are calculated in clockwise order.
+The corners are calculated in the clockwise order.
```

Comment:
synes at the lyder meget mærkeligt i den her sammenhæng

Reply:
måske 'a'

Reply:
er ret sikker på at det er grammatisk korrekt though